src/Signature/SignatureV4.php

```diff
@@ -82,6 +82,7 @@ public function presign(
         $expires,
         array $options = []
     ) {
+        $expiresTimestamp = $this->convertToTimestamp($expires);
```

Comment:
You'd technically get a different duration however it won't throw an error.  Assuming that server times are synced up noone could request it in historical time (requiring the extra second of duration).    The goal of this PR is to fix the opposite case where the second rolls and you fall past the 1 week allow duration and it throws an error, even though the initial request is valid.